CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+
+cmake_minimum_required (VERSION 2.6)
+project (LIBUAST C CXX)
+
+set (CMAKE_CXX_STANDARD 11)
+
+find_package(ZLIB REQUIRED)
```

Comment:
both `zlib` and `lzma` are dependencies of `libxml2`, not ours, I'd say we shouldn't manage them directly